### PR TITLE
Rename to "Deployment Best Practices".

### PR DESCRIPTION
**Pull request by pattivacek**
I have not changed any URLs.

Arguably depends on uptane/deployment-considerations#58.

### CODE REVIEW

**Comment by jhdalek55** [comment]
@pattivacek  Is this a loose thread we need to fix before we send out V.1.1.0? It's a separate issue from the actual document, but we do want to strive for consistency.

**Comment by pattivacek** [comment]
I think this is pretty uncontroversial; it's just following the naming consensus with a couple very minor other changes. I think it can safely be merged now unless you or others disagree.

File: participate.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ using our GitHub [issue tracker](https://github.com/uptane/uptane/issues).
 We welcome security audits of the Uptane design, or vulnerability reports of
 the design or the code in this repository. Please contact lab director
 [Justin Cappos](mailto:jcappos@nyu.edu) or maintainer [Sebastien Awwad](mailto: sebastien.awwad@nyu.edu).
-Uptane design and implementation is defined in the [Uptane Standards document](https://uptane.github.io/uptane-standard/uptane-standard.html). A set of deployment guidelines is being prepared and will be added to this web site upon completion. These deployment guidelines should be viewed as complementary to the official Uptane standard.
+Uptane design and implementation is defined in the [Uptane Standards document](https://uptane.github.io/uptane-standard/uptane-standard.html) and supplmented by the [Deployment Best Practices](https://uptane.github.io/deployment/introduction.html).
```
Comment (jhdalek55) [inline participate.md:60]: supplemented is misspelled

File: reference_pouf.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ This POUF uses ASN.1/DER encoding for transmitting files. The details of the fil
 
 The Primary ECU communicates with the Director repository, Image repository, and time server using HTTP POST requests over [XML-RPC](http://xmlrpc.scripting.com/spec.html). In this communication, the Primary ECU acts as an XML-RPC client and the Director repository, Image repository, and time server act as XML-RPC servers. Secondary ECUs communicate with a Primary ECU using HTTP POST requests over XML-RPC. The Primary acts as the XML-RPC server and the Secondary as an XML-RPC client. The requests supported by all XML-RPC servers in this POUF are described in the Message Handler Table.
 
-All XML-RPC traffic supported in this POUF is transmitted over TCP/IP and does not support CAN or other network types. For more information about internal vehicle communication, see the [Deployment Considerations](https://uptane.github.io/deployment/introduction.html).
+All XML-RPC traffic supported in this POUF is transmitted over TCP/IP and does not support CAN or other network types. For more information about internal vehicle communication, see the [Deployment Best Practices](https://uptane.github.io/deployment/introduction.html).
```
Comment (jhdalek55) [inline reference_pouf.md:22]: change last line to  "...see the Deployment Best Practices document"

File: reference_pouf.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ Of course, as the system implements Uptane, the reference implementation follows
...
+To access a reliable current time as needed for the [In Vehicle Implementation Requirements](https://uptane.github.io/uptane-standard/uptane-standard.html#in-vehicle-implementation-requirements), ECUs in this POUF use a time server that produces a time attestation for each update cycle. The Primary ECU sends the time server nonce tokens from each ECU, then the time server returns a time attestation that contains a signed the current time with all the tokens, as described in the [Deployment Best Practices](https://uptane.github.io/deployment/introduction.html). The ASN.1 details of the time server are listed in {Formats}. The time server is used by the reference implementation as a self contained way to ensure secure access to the current time.
```
Comment (jhdalek55) [inline reference_pouf.md:49]: "the Deployment Best Practices document.

**Review by jhdalek55 — CHANGES_REQUESTED** [review]
I don't know if these have already been fixed in a previous pass. And there was one typos. If this is fixed I will merge.

**Comment by pattivacek** [comment]
Fixed all issues.

**Comment by jhdalek55** [comment]
Done. Thanks.

**Pull request merged**